src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1485,9 +1485,10 @@ public String toStringWithoutGenotypes() {
 
         boolean sawRef = false;
         for ( final Allele a : alleles ) {
```

Comment:
should this be made into a doubly indexed array? wouldn't that be faster?

Reply:
It would, but the input to that method is a collection, not a list so I don't actually have the means to access it by index and I don't want to change the VariantContext API to only accept a list. Getting rid of the iteration over the second list seems to have accounted for ~half of the runtime this method was taking up.